tests/unit/models/transactions/test_deposit_preauth.py

```diff
@@ -26,3 +27,110 @@ def test_authorize_unauthorize_neither_set(self):
                 fee=_FEE,
                 sequence=_SEQUENCE,
             )
+
+    # unit tests using (un)/authorize_credentials input parameter
+    sample_credentials = [
+        Credential(issuer="SampleIssuer", credential_type="SampleCredType")
+    ]
+
+    def test_invalid_input_with_authorize_credentials(self):
+        with self.assertRaises(XRPLModelException):
+            DepositPreauth(
+                account=_ACCOUNT,
+                fee=_FEE,
+                sequence=_SEQUENCE,
+                authorize="authorize",
+                authorize_credentials=self.sample_credentials,
+            )
+
+    def test_valid_input_unauthorize_credentials(self):
+        tx = DepositPreauth(
+            account=_ACCOUNT,
+            fee=_FEE,
+            sequence=_SEQUENCE,
+            unauthorize_credentials=self.sample_credentials,
+        )
+        self.assertTrue(tx.is_valid())
+
+    # Unit tests validating the length of array inputs
+    def test_authcreds_array_input_exceed_length_check(self):
+        # Note: If credentials de-duplication is implemented in the client library,
+        # additional tests need to be written
```

Comment:
This comment is more confusing than helpful IMO

Reply:
fixed in a6ac0f47